mountpoint-s3-client/tests/get_object.rs

```diff
@@ -465,3 +466,88 @@ async fn test_get_object_user_metadata_after_stream(size: usize, metadata: HashM
         .expect("should return metadata");
     assert_eq!(actual_metadata, metadata);
 }
+
+#[test_case(ChecksumAlgorithm::Crc32; "Checksum CRC32")]
```

Comment:
nit: do you need the explicit label? isn't the enum enough to build a meaningful test case name?